src/components/NewFeatures/NewFeatures.js

```diff
@@ -19,13 +19,19 @@ const NewFeatures = () => {
         </div>
       </HeaderSection>
       <section className="dp-container">
+        <div className="dp-block-wlp dp-box-shadow m-t-36">
+          <div style={{ marginLeft: '20px' }}>
+            <h3>Dashboard</h3>
+            <Link to={'/dashboard'}>Ir a Dashboard</Link>
+          </div>
+        </div>
         <div className="dp-block-wlp dp-box-shadow m-t-36">
           <div style={{ marginLeft: '20px' }}>
             <h3>Compra de planes</h3>
             <Link to={'/plan-selection?promo-code=ALLPLANS'}>Ir a Comprar Plan</Link>
           </div>
         </div>
-        <div className="dp-block-wlp dp-box-shadow m-t-36 m-b-36">
+        <div className="dp-block-wlp dp-box-shadow m-t-36">
```

Comment:
Because I saw that it did not modify and it was the only one that had that class. And I added it to the last element because it was stuck to the footer. But if it's wrong I undo it

Reply:
I've seen it working and it looks better as you said so well done!